flow/texture_unittests.cc

```diff
@@ -0,0 +1,43 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/flow/texture.h"
+#include "gtest/gtest.h"
+
+namespace flutter {
+namespace testing {
+
+class MockTexture : public Texture {
+ public:
+  MockTexture(int64_t textureId) : Texture(textureId) {}
+
+  ~MockTexture() override = default;
+
+  bool unregistered = false;
```

Comment:
Per naming convention, this needs to be private and have an underscore suffix. You can add a getter to check the assertion.